docs-source/content/userguide/managing-domains/domain-lifecycle/scaling.md

```diff
@@ -206,8 +206,8 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: weblogic-domain-cluster-role
 rules:
-- apiGroups: ["weblogic.oracle"]
-  resources: ["domains"]
+- apiGroups: ["apiextensions.k8s.io", "weblogic.oracle"]
+  resources: ["customresourcedefinitions", "domains"]
```

Comment:
Let me be clearer...

Current proposal:

rules:
- apiGroups: ["apiextensions.k8s.io", "weblogic.oracle"]
  resources: ["customresourcedefinitions", "domains"]
  verbs: ["get", "list", "update"]

This isn't acceptable because it gives the scaling user, "customresourcedefinitions" / "update", which is too much privilege.

Instead, separate into two rules with only the necessary verbs:

rules:
- apiGroups: ["weblogic.oracle"]
  resources: ["domains"]
  verbs: ["get", "list", "update"]
- apiGroups: ["apiextensions.k8s.io"]
  resources: ["customresourcedefinitions"]
  verbs: ["get", "list"]

Reply:
sounds good. I will test with these changes and get back.

Reply:
Sorry, I didn't realize I was added to this review and I would agree with Ryan about not giving update permission to a CRD in kubernetes cluster.

@rjeberhard Ryan, I'd also like to point out that the RBAC permission to update the domain resource across any namespace in cluster for REST API is pointed out in JIRA OWLS-62312.

However this PR to document the requirement for `scalingAction.sh` is fine as far as I can tell...

Thanks, -Craig

Reply:
This is verified. @rosemarymarano you can update with the below for rules..
```
rules:
- apiGroups: ["weblogic.oracle"]
  resources: ["domains"]
  verbs: ["get", "list", "update"]
- apiGroups: ["apiextensions.k8s.io"]
  resources: ["customresourcedefinitions"]
  verbs: ["get", "list"]
```